packages/cfd/src/Containers/cfd-password-modal.tsx

```diff
@@ -534,26 +537,34 @@ const CFDPasswordForm = ({
                                 )}
                             </Text>
                         )}
+                        {/* TODO: Check if do we have to add 'PasswordMeter' in production aswell. */}
                         <div className='input-element'>
-                            <PasswordInput
-                                autoComplete='new-password'
-                                label={localize('{{platform}} password', {
-                                    platform: getCFDPlatformLabel(platform),
-                                })}
-                                error={
-                                    (touched.password && errors.password) ||
-                                    (values.password.length === 0 ? error_message : '')
-                                }
-                                name='password'
-                                value={values.password}
-                                onBlur={handleBlur}
-                                onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
-                                    handlePasswordInputChange(e, handleChange, validateForm, setFieldTouched);
-                                }}
-                                data_testId={`dt_${platform}_password`}
-                            />
+                            <PasswordMeter
+                                input={values.password}
+                                has_error={!!(touched.password && errors.password)}
+                                custom_feedback_messages={getErrorMessages().password_warnings}
+                            >
+                                {() => (
```

Comment:
Why are we passing the children in a function ?
we aren't using any parameters.

Reply:
My bad I overlook this thanks for pointing it out 👍🏼